src/foam/comics/DAOControllerView.js

```diff
@@ -80,12 +83,14 @@ foam.CLASS({
             start('td').style({'vertical-align': 'top', 'width': '100%'}).
               start('span').
                 style({background: 'rgba(0,0,0,0)'}).
-                show(this.mode$.map(function(m) { return m == foam.u2.DisplayMode.RW; })).
-                start().
-                  style({padding: '4px 4px 4px 1px'}).
-                  add(this.cls.getAxiomsByClass(foam.core.Action)).
-                end().
-              end().
+                show(self.mode$.map(function(m) { return m == foam.u2.DisplayMode.RW; })).
+                callIf(this.window.location.hash != '#allInvoices', function() {
```

Comment:
This is not a sustainable way to customize. Doing this will eventually litter this otherwise generic code with many application specific exceptions. Better to make a 'showActions' property and then just configure the allInvoices one differently, or else just add a class to this section and then use CSS to hide it.

Reply:
This references work done in our private repo so I don't think it should be added here.